testsuite/tests/tool-ocamldoc-2/inline_records.reference

```diff
@@ -50,14 +76,114 @@ A simple record type for reference
 
 \label{TYPInline-underscorerecords.t}\begin{ocamldoccode}
 type t =
-  | C of {\char123}
-   lbl : int;
-   more : int list;
-{\char125}
+
 \end{ocamldoccode}
```

Comment:
It also looks like the field documentations are actually not processed. Is that a mistake on my part reading the output?